### PR TITLE
Material UI Time Picker

### PR DESCRIPTION
**Pull request by akgupta89**
Working on converting ControlPanel to a functional component and replacing the date time picker with a material UI dropdown

Preview:
Screen Shot 2019-07-17 at 6 49 03 PM

Feel free to leave comments / suggestions

### CODE REVIEW

**Comment by hathix** [comment]
Can you deploy a demo on Heroku? Terence did something similar

File: frontend/src/components/ControlPanel.jsx [hunk 1]
```diff
@@ -53,218 +118,215 @@ class ControlPanel extends Component {
...
-         `}
-          />  */ }
+  return (
+    <div
+      className={css`
+        color: #fff;
+        border-radius: 5px;
+        padding: 10px;
+        margin-right: 20px;
+        grid-column: col1-start / col3-start;
```
Comment (hathix) [inline frontend/src/components/ControlPanel.jsx]: This is a lot of inline styling. Can we apply a class and move the styling into CSS?

File: frontend/src/components/Navigation.jsx [hunk 2]
```diff
@@ -7,54 +7,53 @@ export default props => {
...
-      <NavLink
-        to={{ type: 'ROUTESCREEN' }}
-        activeStyle={{ fontWeight: "bold", color: 'purple' }}
-        exact={true}
-        strict={true}
-      >
-        Route
+        <NavLink
+          to={{ type: 'ROUTESCREEN' }}
+          activeStyle={{ fontWeight: 'bold', color: 'purple' }}
```
Comment (hathix) [inline frontend/src/components/Navigation.jsx]: could we pull out the purple color into CSS or some constant? I imagine we'll want to change the color scheme later.

Comment (akgupta89) [reply frontend/src/components/Navigation.jsx]: We should open a separate PR / ticket for this

**Comment by akgupta89** [comment]
Basically the from / to select dropdowns need to be fixed when selected which is a small bug, but everything else works fine. I'm gone next week so if someone wants to finish this feel free.

File: frontend/src/components/Navigation.jsx (hunk 2 above)
Comment (hathix) [reply frontend/src/components/Navigation.jsx]: In #209

**Comment by hathix** [comment]
Looks like this is ready to review!

File: frontend/src/components/ControlPanel.jsx [hunk 3]
```diff
@@ -1,270 +1,338 @@
...
+                          (
+                            selectedRoute.stops[firstStopId] || {
+                              title: firstStopId,
+                            }
+                          ).title
+                        }
+                      </MenuItem>
+                    ))}
+                  </Select>
+                  <FormHelperText>Some important helper text</FormHelperText>
```
Comment (hathix) [inline frontend/src/components/ControlPanel.jsx]: does this need to be replaced with real text?

File: frontend/src/components/ControlPanel.jsx [hunk 4]
```diff
@@ -1,270 +1,338 @@
...
-        </Card>
-      </div>
-    );
-  }
+          ) : null}
+          {selectedDirection ? (
+            <div className="dropDownOverlay">
+              <ListGroup.Item>
+                <FormControl className={classes.formControl}>
+                  <InputLabel htmlFor="to-stop-helper">To Step</InputLabel>
```
Comment (exxonvaldez) [inline frontend/src/components/ControlPanel.jsx]: Typo, change step -> stop.

File: frontend/src/components/ControlPanel.jsx (hunk 1 above)
Comment (hathix) [reply frontend/src/components/ControlPanel.jsx]: has the new css class been added?

File: frontend/src/components/ControlPanel.jsx [hunk 5]
```diff
@@ -1,270 +1,338 @@
...
+              <MenuItem value="07:00-19:00">Daytime (7AM - 7PM)</MenuItem>
+              <MenuItem value="03:00-07:00">Early Morning (3AM - 7AM)</MenuItem>
+              <MenuItem value="07:00-10:00">AM Peak (7AM - 10AM)</MenuItem>
+              <MenuItem value="10:00-15:00">Midday (10AM - 4PM)</MenuItem>
+              <MenuItem value="16:00-19:00">PM Peak (4PM - 7PM)</MenuItem>
+              <MenuItem value="19:00-03:00+1">
+                Late Evening (7PM - 3AM)
+              </MenuItem>
+            </Select>
+            <FormHelperText>Some important helper text</FormHelperText>
```
Comment (hathix) [inline frontend/src/components/ControlPanel.jsx]: should this be replaced with real text?

File: frontend/src/components/ControlPanel.jsx [hunk 6]
```diff
@@ -1,270 +1,338 @@
...
-                  value={graphParams.route_id}
-                  options={
-                    (routes || []).map(route => ({
-                      label: route.title, key: route.id,
-                    }))
-                  }
-                  onSelect={this.setRouteId}
-                />
+                <FormControl className={classes.formControl}>
+                  <InputLabel htmlFor="from-stop-helper">From Step</InputLabel>
```
Comment (exxonvaldez) [inline frontend/src/components/ControlPanel.jsx]: Typo step -> stop.

**Pull request merged**